app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
I've not dived into it yet, but I think the HTML is going to be moving around. For example, on Detailed Guides, we need to shift the content to the left two thirds to make room for a Related Items box on the right. This would include shifting the contents and some metadata out of the way (presumably to the top of the document?). I'm not sure that just picking different partials will really cut it in this case.